### ISSUE TITLE
Russian eSpeak

### ISSUE THREAD

**Issue by Jonikster** [espeak-ng/espeak-ng#1503]
The latest changes to eSpeak regarding the Russian language are definitely worse than they used to be.
This is especially true for the numbers 8 and 80. The pronunciation that was used before may not be perfect, but now it sounds worse.
I suggest cancel many of the changes, I just didn't find certain commits regarding these things.

**Comment by IvLeb474** [comment]
Just checked, it's definitely not good.
I'm not sure if there's a Russian speakers here, just check, old eSpeak sounded better, but I use RHVoice though.

**Comment by kirill-jjj** [comment]
@IvLeb474, Joined GitHub
on November 4, 2022, not suspicious at all

**Comment by Jonikster** [comment]
@kirill-jjj do you know other ways to post messages here?

**Comment by cyrmax** [comment]
Fortunately, ESpeak tries to sound like human voice as much as posible. 
For me ESpeak with recent changes sounds much more humanlike than before. Yes, there are still some phonetic problems but now it looks more usable especialy for me.

**Comment by cyrmax** [comment]
@Jonikster @kirill-jjj 
Please do stick to the issue topic. I think no offtopic should be here on Github.

**Comment by Jonikster** [comment]
@cyrmax did you hear how like 80 sound?

**Comment by cyrmax** [comment]
@Jonikster yep i know how it sounds and for me it sounds much better than before. 
ESpeak comes more and more human sounding.

**Comment by alex19EP** [comment]
I understand that we are all different people, and for everyone the sound of espeak-ng will be subjective, but I agree with the statements of the people above. at the moment, the numbers 8 and 80 sound much better than before.

@Jonikster maybe you can suggest a better sound, but it definitely shouldn't be what it used to be before.
